aten/src/ATen/native/RNN.cpp

```diff
@@ -917,6 +919,7 @@ struct FullBidirectionalLayer
       step_inputs = input_w.unbind(0);
       auto fw_result = layer_(
           step_inputs, input_hidden.first, params.first, true);
+      TORCH_CHECK(fw_result.outputs.size() > 0, "Expected sequence length to be larger than 0 in RNN");
```

Comment:
can you add some tests to test/test_nn.py that try out some error cases and assert that they raise the warning?

In particular we want to check that for all combinations of: {nn.LSTM, nn.GRU, nn.RNN} x {bidirectional, unidirectional} x {device cpu, device cuda}, a input with seq_len=0 will raise the error message. 

The CUDA case is interesting because I'm not sure if the CUDA RNNs go through a different code path or not... but we will find out.

Reply:
tests are added :) thanks for the feedback